adapter_docs/loading.md

```diff
@@ -20,6 +20,16 @@ for adapter_info in adapter_infos:
     print("Uploaded by:", adapter_info.username)
 ```
 
+In case the adapter ID is known, information for a single adapter can also be retrieved via [`get_adapter_info()`](classes/adapter_utils.html#transformers.adapters.utils.get_adapter_info):
+
+```python
+adapter_info = get_adapter_info("@ukp/bert-base-uncased_sentiment_sst-2_pfeiffer", source="ah")
+
+print("Id:", adapter_info.adapter_id)
+print("Model name:", adapter_info.model_name)
+print("Uploaded by:", adapter_info.username)
```

Comment:
Maybe we could link the Documentation for the [Adapter Info class](https://docs.adapterhub.ml/classes/adapter_utils.html?highlight=adapterinfo#transformers.adapters.utils.AdapterInfo) as well and add a description for all parameters. At the moment only adapter_id, model_name, and source are described in detail.

Reply:
Thanks, done :)